src/BuildScriptGenerator/Node/NodeConstants.cs

```diff
@@ -34,7 +34,7 @@ public static class NodeConstants
         public const string YarnVersionCommand = "echo Using Yarn version: && yarn --version";
         public const string HugoVersionCommand = "echo Using Hugo version: && hugo version";
         public const string YarnPackageInstallCommand = "yarn install --prefer-offline";
-        public const string Yarn2PackageInstallCommand = "yarn plugin import workspace-tools && yarn workspaces focus --all";
+        public const string Yarn2PackageInstallCommand = "yarn workspaces focus --all || yarn install";
```

Comment:
`yarn plugin import workspace-tools` I think this is the plugin required for running `yarn workspaces focus --all`, if we remove it we may not able to run the later command.

Reply:
@qianz2  ... so basicaly you are kind of true but the whole scenario/workflow is little different ... here is a good reference https://github.com/yarnpkg/berry/issues/3199

so based on this, I think "yarn plugin import workspace-tools" is usually gets executed only once, it should not run every time .. user should run it once and check the plugin in in their repo and from next time onwards it should be only yarn workspace focus --all (as the workspace is already been set). So assumption is this command is basically user's responsibility and oryx should not do it.

If the workspace is not set yarn workspace focus --all will fail but then we will run yarn install command for yarn2.

Reply:
@qianz2  you can look at the test that I added to get more insight ..  let me know if you have some questions

Reply:
I see, then we will let users run this plugin command, thanks for the explain. And we can include this into oryx docs.